src/components/shared/card/index.tsx

```diff
@@ -96,21 +98,23 @@ const Card = ({
           )}
         </div>
 
-        <div className='flex min-w-0 flex-1 flex-col'>
-          <h3 className='text-text-md-semibold w-full truncate text-gray-900'>{title}</h3>
+        <div className='flex min-w-0 flex-1 flex-col justify-between'>
```

Comment:
```suggestion
        <div className='flex-col-between min-w-0 flex-1'>
```
여기두 제가 만든거 쓰면 훨신 짧아지쥬?